src/Mobs/Spider.cpp

```diff
@@ -43,9 +43,15 @@ void cSpider::EventSeePlayer(cEntity * a_Entity)
 		return;
```

Comment:
I tried a bailout without a queue. Nothing changed in-game. I guess I'll go for that. In the very worst case scenario, a spider will not attack a player in some edge case even though it's dark. This is not worth worrying about.